### PR TITLE
OpenGEE Blog - Foss4G Blog Post

### PR DESCRIPTION
**Pull request by tst-ahernandez**
Added a new entry to the blog for the recent FOSS4G event.

Preview of the entry here: [preview](https://tst-ahernandez.github.io/earthenterprise/blog.html)

### CODE REVIEW

**Comment by tst-lsavoie** [comment]
Any reason why there are so many commits here?  It would be nice to clean those up.

**Comment by tst-ccamp** [comment]
I think we should update upload our slides and link to them from the post.

**Comment by tst-ccamp** [comment]
@tst-ahernandez @tst-lsavoie I think as long as we "Squash and merge" we can get rid of all these extra commits. I wonder if maybe a lot of the merge commits are due to the branch needing to be rebased. You can usually fix that pretty easily with:
```
git fetch upstream
git rebase upstream/master
git push origin --force
```

File: docs/_posts/2017-08-31-ahernandez-foss.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: blogpost
+title:  "2017 FOSS 4G"
```
Comment (tst-lsavoie) [inline docs/_posts/2017-08-31-ahernandez-foss.md]: Minor: "FOSS 4G" or "FOSS4G" - with or without space?

**Comment by tst-ahernandez** [comment]
@tst-ccamp The presentation slides are still in post-edit and not ready to upload yet. Do we want to wait on this post until that time?

**Comment by tst-ahernandez** [comment]
Approvals are in!

**Pull request merged**